packages/mask/src/extension/options-page/DashboardComponents/CollectibleList/CollectibleItem.tsx

```diff
@@ -0,0 +1,82 @@
+import type { Web3Helper } from '@masknet/plugin-infra/web3'
+import { makeStyles, MaskColorVar } from '@masknet/theme'
+import type { NetworkPluginID, NonFungibleAsset, SocialAddress, SourceType, Wallet } from '@masknet/web3-shared-base'
+import { Skeleton, Typography } from '@mui/material'
+import type { HTMLProps } from 'react'
+import { CollectibleCard } from './CollectibleCard'
+
+const useStyles = makeStyles()((theme) => ({
+    card: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        position: 'relative',
+        padding: theme.spacing(1, 0),
+    },
+    collectibleCard: {
+        width: '100%',
+        height: '100%',
+        aspectRatio: '1/1',
+    },
+    description: {
+        background: theme.palette.mode === 'light' ? MaskColorVar.twitterBackground : MaskColorVar.twitterLine,
```

Comment:
I think the MaskColorVar is deprecated. CC @nuanyang233

Reply:
It's deprecated, but it's also hard to tell what color can match. There are still many usages in the codebase.